test/helpers.ts

```diff
@@ -0,0 +1,58 @@
+import { BigNumberish } from "ethers";
+import { network, ethers } from "hardhat";
+
+export const getBigNumber = (num: any) => ethers.BigNumber.from(num);
+
+export const getTimeStamp = (date: number) => Math.floor(date / 1000);
+
+export const getTimeStampNow = () => Math.floor(Date.now() / 1000);
+
+export const getDate = (timestamp: number) => new Date(timestamp * 1000).toDateString();
+
+export const getSeconds = (days: number) => 3600 * 24 * days; // Changes days to seconds
+
+export const impersonateAccounts = async (accounts: string | any[]) => {
+    let signers = [];
+
+    for (let i = 0; i < accounts.length; ++i) {
+        await network.provider.request({
+            method: "hardhat_impersonateAccount",
+            params: [accounts[i]]
+        });
+
+        await network.provider.send("hardhat_setBalance", [
+            accounts[i],
+            ethers.utils.hexValue(1000),
+            // hexValue(parseEther("1000")),
```

Comment:
```suggestion
```

Reply:
This isn't really the scope of this PR. This PR is strictly related to `REXSushiFarmMarket.sol` and it's dependencies.